backend/src/main/resources/application.properties

```diff
@@ -102,4 +102,5 @@ quarkus.unleash.active=false
 quarkus.unleash.url=http://localhost:4242
 
 # Kessel
-notifications.kessel.target-url=localhost:9000
+notifications.kessel.inventory-api.target-url=localhost:9080
+notifications.kessel.relationships-api.target-url=localhost:9000
```

Comment:
That's nice! The next PR also refactors this so that we use the client libraries' configuration values directly, because it's just easier to use their config for the providers.